src/pf-touchspin/pf-touchspin.component.js

```diff
@@ -0,0 +1,359 @@
+/**
+ * <b>&lt;pf-touchspin&gt;</b> element for Patternfly Web Components
+ *
+ * <pf-touchspin id="touchspin" class="input-group bootstrap-touchspin" decimals="2" step="0.1">
+ *  <span class="input-group-btn">
+ *    <button class="btn btn-default bootstrap-touchspin-down" type="button">-</button>
+ *  </span>
+ *  <input value="50.00" type="text" class="form-control">
+ *  <span class="input-group-btn">
+ *    <button class="btn btn-default bootstrap-touchspin-up" type="button">+</button>
+ *  </span>
+ * </pf-touchspin>
+ *
+ * @prop {number} min the minimum value
+ * @prop {number} max the maximum value
+ * @prop {number} step Increment/Decrement in value on up/down
+ * @prop {number} decimals decimal points in value
+ * @prop {boolean} booster if true, spinner will become faster continousally on holding down the button
+ * @prop {number} boostat boost at every nth step
+ * @prop {number} maxboostedstep maximum step when boosted
+ * @prop {number} stepinterval refresh reate of spinner in millisecond
+ * @prop {number} stepintervaldelay delay before sppiner starts to spin(millisecond)
+ * @prop {string} forcestepdivisibility force the value to be divisible by step value: 'none' | 'round' | 'floor' | 'ceil'
+ *
+ */
+
+export class PfTouchspin extends HTMLElement {
+
+  init() {
+    this._min = parseFloat(this.getAttribute('min')) ? this.getAttribute('min') : 0;
+    this._max = parseFloat(this.getAttribute('max')) ? this.getAttribute('max') : 100;
+    this._step = parseFloat(this.getAttribute('step')) ? this.getAttribute('step') : 1;
+    this._decimals = parseInt(this.getAttribute('decimals')) ? this.getAttribute('decimals') : 0;
+    this._booster = this.getAttribute('booster') ? this.getAttribute('booster') : true;
+    this._boostat = parseInt(this.getAttribute('boostat')) ? this.getAttribute('boostat') : 10;
+    this._maxBoostedStep = this.getAttribute('maxboostedstep') ? this.getAttribute('maxboostedstep') : false;
+    this._stepInterval = this.getAttribute('stepinterval') ? this.getAttribute('stepinterval') : 100;
+    this._stepIntervalDelay = this.getAttribute('stepintervaldelay') ? this.getAttribute('stepintervaldelay') : 500;
+    this._forceStepDivisibility = this.getAttribute('forcestepdivisibility') ? this.getAttribute('forcestepdivisibility') : 'round';
+    this._spinning = false;
+    this.spincount = 0;
+  }
+
+  connectedCallback() {
+    let self = this;
+    var input = this.querySelector('input');
```

Comment:
can we use let or const for variables across the board? this is minor, but just prefer to not use "var" since we have es6 now ;)